edg_core/Blocks.py

```diff
@@ -290,6 +326,13 @@ def _populate_def_proto_block_contents(self, pb: BaseBlockEdgirType) -> BaseBloc
 
     return pb
 
+  def _populate_def_proto_description(self, pb: BaseBlockEdgirType) -> BaseBlockEdgirType:
+    description = self.description
+    if isinstance(description, DescriptionString):
```

Comment:
I'd prefer an assert here, generally I'd prefer to fail noisily on bad input (so the user at least knows something is wrong) than silently discarding data.